core/modfunc.go

```diff
@@ -197,18 +195,6 @@ func (fn *ModuleFunction) Call(ctx context.Context, opts *CallOpts) (t dagql.Typ
 		return nil, fmt.Errorf("failed to mount mod metadata directory: %w", err)
 	}
 
-	// Used by the shim to associate logs to the function call instead of the
-	// exec /runtime process, which we hide.
-	tc := propagation.TraceContext{}
-	carrier := propagation.MapCarrier{}
-	tc.Inject(ctx, carrier)
-	for _, f := range tc.Fields() {
-		name := "DAGGER_FUNCTION_" + strcase.ToScreamingSnake(f)
-		if val, ok := carrier[f]; ok {
-			callMeta.OTELEnvs = append(callMeta.OTELEnvs, name+"="+val)
-		}
-	}
-
```

Comment:
see https://github.com/dagger/dagger/pull/7452/files#r1628294096 - correction: we were setting it, but nothing reads this env var anymore